src/main/scala/com/udemy/statistics/algorithm/Sampling.scala

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2018 Udemy, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.udemy.statistics.algorithm
+
+import scala.Numeric.Implicits._
+import scala.collection.parallel.immutable.ParVector
+import scala.util.Random
+import scalaz.NonEmptyList
+import scalaz.Scalaz._
+
+object Sampling {
+
+  def randomDraw[T:Numeric](x: NonEmptyList[T]): T =
```

Comment:
<sup>nit</sup>: `T: Numeric` (with the space) for consistency?

Reply:
Ah, thanks. Not sure how these ended up inconsistent.

Reply:
fixed